kyo-core/shared/src/main/scala/kyo/logs.scala

```diff
@@ -17,43 +17,43 @@ object Logs:
         def errorEnabled: Boolean
 
         def trace(msg: => String)(
-            using file: internal.FileNameWithLine
+            using file: internal.Position
```

Comment:
I intentionally left this named `file` - I think it reads well as 'file position'